spring-cloud-contract-verifier/src/main/java/org/springframework/cloud/contract/verifier/messaging/kafka/KafkaStubMessages.java

```diff
@@ -87,85 +87,80 @@ public Message receive(String destination) {
 
 	@Override
 	public void send(Object payload, Map headers, String destination) {
-		Message<?> message = MessageBuilder.createMessage(payload,
-				new MessageHeaders(headers));
+		Message<?> message = MessageBuilder.createMessage(payload, new MessageHeaders(headers));
 		send(message, destination);
 	}
 
 }
 
 class Receiver {
 
-	private final Map<String, Consumer> consumers;
-
 	private static final Log log = LogFactory.getLog(Receiver.class);
 
+	private final MessagingMessageConverter messagingMessageConverter = new MessagingMessageConverter();
```

Comment:
We delegate to spring-kafka to set all of the "special" headers for us. This future proofs when the next issue is files that says "I can't access the Spring Kafka message header ...".